src/ifs_interface/ifs_interface.F90

```diff
@@ -862,24 +818,20 @@ SUBROUTINE nemogcmcoup_lim2_update( mype, npes, icomm, &
    ! Sort out incoming arrays from the IFS and put them on the ocean grid
 
    ! TODO
-   shortwave(:)=0.	! Done, updated below. What to do with shortwave over ice??
```

Comment:
@trackow is this all solved as well?

Reply:
Yes, there are no open questions really in the coupling. Just make sure that the resetting to 0 is still happening here, the comments can go